packages/cow-hooks-ui/src/withdraw/PoolBalancePreview.tsx

```diff
@@ -85,7 +85,7 @@ export function PoolBalancesPreview({
                         balance.token.decimals,
                       ),
                     )}
-                    fiatValue={balance.fiatAmount}
+                    fiatValue={withdrawBalance[index].fiatAmount}
```

Comment:
Is this another bug fix? 🤔

Reply:
yep, before this modification, changing the withdraw percentage was not updating the displayed fiat amount